docs_app/content/tutorial/1-subscribing.md

```diff
@@ -0,0 +1,66 @@
+
+## Overview
+
+Chances are, the first time you encounter an [Observable](API), it is because you have had one returned to you by some other API. In this case, the most helpful thing for you to know is how to get values from the observable, and the semantics of [subscription](GL)
+
+> The first thing to know is, an observable doesn't "do" anything.
+
+An observable, in fact, is _not_ a "stream". An observable is a template for [subscription](GL), through which your code (the [consumer](GL)), will tell a [producer](GL) to start pushing values to it by subscribing to an observable, creating a [subscription](GL).
+
+## Basic Subscription
+
+```ts
+const source$ = getSomeObservable();
+
+source$.subscribe(value => {
+    // do something with each value as it is pushed to your code
+    console.log(value);
+});
+```
+
+The most basic form of subscription is to subscribe to the observable with a single function as the argument to [`subscribe`](API). This will execute the observable's initialization function and create the [subscription](GL).
+
+## Subscription With An Observer
+
+```ts
+const source$ = getSomeObservable();
+
+source$.subscribe({
+    next(value) {
+        // Do something with each value as it is pushed to your code
+        console.log(value);
+    },
+    error(err) {
+        // Whatever is producing and pushing values has encountered an error
+        // and will no longer send values
+        console.error(err);
+    },
+    complete() {
+        // The producer pushing values is done pushing values, and wishes to notify
+        // your code that you will no longer receive values.
+        console.log('done');
+    }
+})
+```
+
+An `Observer` is a type, basically any object with a `next`, `error`, and `complete` method on it. A "partial observer" is any object with at least one of those methods. It is a way to pass these three handlers to the subscription so that a [producer](GL) can notify your code (the [consumer](GL)). `next` is called when the producer pushes a value to the consumer. `error` is called when the producer has encountered an error and must stop sending values, and `complete` is called  when the producer has pushed all values to the consumer, and will push no more.
+
+
+## Unhandled Errors
+
+If you call subscribe with one function ([Basic Subscription](#Basic_Subscription)), or with a partial [observer](#Subscription_With_An_Observer) that does not have an [error](GL) handler, RxJS will treat all errors pushed by the [producer](GL) as "unhandled". This means that the developer did nothing to handle the error.
+
+All unhandled errors will be rethrown in a different call context. This is done for a variety of reasons, but what is important to know is: Even if your observable is totally synchronous, an error from your observable _cannot be caught_ by wrapping the [`subscribe`](API) call in a `try-catch` block.
+
+
+### Handled Errors
```

Comment:
Should this be ## instead of ### so it's at the same level as "Unhandled Errors"?